lib/index.js

```diff
@@ -0,0 +1,126 @@
+const fs = require('fs');
+const {
+  fetchLabels,
+  createLabels,
+  deleteLabels,
+  updateLabels,
+} = require('./api');
+
+async function labelControl() {
+  const labelData = await fetchLabels();
+  const formattedPublishedLabels = await labelFormatter(labelData);
+  const data = fs.readFileSync('./label-config.json', { encoding: 'utf8' });
```

Comment:
Can you open a new issue to make the config configurable?

Most js libraries use this library to help with that: https://github.com/davidtheclark/cosmiconfig

Reply:
Done 👍 
#5